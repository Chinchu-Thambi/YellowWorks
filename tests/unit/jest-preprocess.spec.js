/* globals describe, it, expect */

import config from '../../jest-preprocess';

describe('jest preprocess', () => {
  it('should not change by accident', () => {
    expect(config).toMatchSnapshot();
  });
});
