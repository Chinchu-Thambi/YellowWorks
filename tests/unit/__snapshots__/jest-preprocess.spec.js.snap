// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest preprocess should not change by accident 1`] = `
Object {
  "canInstrument": true,
  "getCacheKey": [Function],
  "process": [Function],
}
`;
